compat/test/browser/portals.test.js

```diff
@@ -452,6 +452,6 @@ describe('Portal', () => {
 
 		toggle();
```

Comment:
For sanity's sake could you duplicate this test with `<Modal open={open}>Hello</Modal>` as firstChild, I remember trying to avoid `append` for some reason if that works out I'll test something myself and else approve the PR

Reply:
Added the test